docs/src/guides/spring/rabbitmq.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Configure Spring with RabbitMQ
+sidebarTitle: RabbitMQ
+weight: -110
+description: Configure a Spring application with RabbitMQ.
+---
+
+You can use [Spring JMS](https://spring.io/guides/gs/messaging-jms/)
+to use [RabbitMQ](../../configuration/services/rabbitmq.md) with your app.
+First, determine the MongoDB client using the [Java configuration reader library](https://github.com/platformsh/config-reader-java).
```

Comment:
This step is fairly vague (also for MySQL, ...).
Reviewing this in-depth would fall out of the scope of the current review. 
Worth adding an issue for future change?

Reply:
All of the guides need to be redone. So I think the issue will be created when it comes up in the priority.

Reply:
That makes sense! Safe to ignore, then.